web/styles/right_sidebar.css

```diff
@@ -49,6 +49,60 @@ $user_status_emoji_width: 24px;
     }
 }
 
+.buddy-list-section li {
```

Comment:
The `li` part here is probably useless and should be deleted, possibly both here and below? But that's not new in this PR.

Reply:
It's needed for the `hover` stuff, but I could move it down to the hover styles specifically if you want.

Reply:
OK but couldn't it be just `.user_sidebar_entry` instead?